force-app/main/default/objects/npe03__Recurring_Donations_Settings__c/fields/EnableChangeHistory__c.field-meta.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
```

Comment:
This should have been part of feature/234, but I guess I missed it during the sfdx conversion 😬